openpype/settings/entities/input_entities.py

```diff
@@ -442,7 +442,9 @@ class TextEntity(InputEntity):
 
     def _item_initialization(self):
         self.valid_value_types = (STRING_TYPE, )
-        self.value_on_not_set = ""
+        self.value_on_not_set = self.convert_to_valid_type(
+            self.schema_data.get("default", "")
+        )
```

Comment:
This shouldn't be in this PR I think?

Reply:
It is necessary for adding defaults to the `text` field. We need defaults else the fields are empty when adding a new profile which is a lot of added admin work.
Do you want it in a separate PR?